lib/native/mqtt.ts

```diff
@@ -419,7 +419,10 @@ export class MqttClientConnection extends NativeResourceMixin(BufferedEventEmitt
     private _reject(reject: (reason: any) => void) {
         return (reason: any) => {
             reject(reason);
-            this.emit('error', new CrtError(reason));
+            let listenerArray = this.rawListeners('error');
```

Comment:
Just add a comment as to why we're doing this, so some poor unsuspecting fool (most likely one of us) doesn't remove it in the future and suffer needlessly